4-WebApp-your-API/TodoListService/Startup.cs

```diff
@@ -51,13 +51,11 @@ public void ConfigureServices(IServiceCollection services)
             // By default, the claims mapping will map claim names in the old format to accommodate older SAML applications.
             // 'http://schemas.microsoft.com/ws/2008/06/identity/claims/role' instead of 'roles'
             // This flag ensures that the ClaimsIdentity claims collection will be built from the claims in the token
-            JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
+            //JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
+
+            services.AddHttpContextAccessor();
```

Comment:
I think that this should be in AddProtectedWebApi, not in Startup.cs ?

Reply:
@TiagoBrenck : on this one, we don't want apps to have to care about implementation details. .AddProtectedWebApi() needs to AddHttpContextAccessor if it uses it (I'm not sure what for, though?)

Reply:
moved to the library.